types/address/codec.go

```diff
@@ -0,0 +1,26 @@
+package address
+
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+type Codec interface {
+	// ConvertAddressStringToBytes decodes text to bytes
+	ConvertAddressStringToBytes(text string) ([]byte, error)
+	// ConvertAddressBytesToString encodes bytes to text
+	ConvertAddressBytesToString(bz []byte) (string, error)
```

Comment:
```suggestion
	AddressStringToBytes(text string) ([]byte, error)
	// ConvertAddressBytesToString encodes bytes to text
	AddressBytesToString(bz []byte) (string, error)
```

Let's make it a bit shorter

Reply:
Since this change will introduce the name conflict detailed [here](https://github.com/cosmos/cosmos-sdk/pull/9759#discussion_r682085847), I would leave the names as it is.

Reply:
any thoughts, @aaronc ?